backend/LexBoxApi/GraphQL/ProjectMutations.cs

```diff
@@ -134,18 +134,17 @@ public record BulkAddProjectMembersResult(List<UserProjectRole> AddedMembers, Li
     [Error<NotFoundException>]
     [Error<InvalidEmailException>]
     [Error<DbError>]
-    [AdminRequired]
     [UseMutationConvention]
     public async Task<BulkAddProjectMembersResult> BulkAddProjectMembers(
+        IPermissionService permissionService,
         LoggedInContext loggedInContext,
         BulkAddProjectMembersInput input,
         LexBoxDbContext dbContext)
     {
-        if (input.ProjectId.HasValue)
-        {
-            var projectExists = await dbContext.Projects.AnyAsync(p => p.Id == input.ProjectId.Value);
-            if (!projectExists) throw new NotFoundException("Project not found", "project");
-        }
+        await permissionService.AssertCanCreateGuestUserInProject(input.ProjectId);
+        if (input.ProjectId == null) throw new NotFoundException("Project not found", "project");
```

Comment:
yeah I don't think this makes sense anymore, just make it not nullable. Looking at some comments lower down we were thinking this mutation would be used from other pages but that's not the case so let's simplify it

Reply:
Done in commit 5ffa7ad7.